pkg/api/handlers/libpod/images.go

```diff
@@ -234,6 +234,76 @@ func ExportImage(w http.ResponseWriter, r *http.Request) {
 	utils.WriteResponse(w, http.StatusOK, rdr)
 }
 
+func ExportImages(w http.ResponseWriter, r *http.Request) {
+	var (
+		output string
+	)
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	query := struct {
+		Compress   bool     `schema:"compress"`
+		Format     string   `schema:"format"`
+		References []string `schema:"references"`
+	}{
+		Format: define.OCIArchive,
+	}
+
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
+		utils.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest,
+			errors.Wrapf(err, "Failed to parse parameters for %s", r.URL.String()))
+		return
+	}
+
+	// References are mandatory!
+	if len(query.References) == 0 {
```

Comment:
You could use .Queries("References", "*") when registering this endpoint. The framework would take care of the validation.  "Format doesn't work as well, since you have a non-Zero default for that parameter.